### ISSUE TITLE
Function used by createActionAsync is only passed one parameter

### ISSUE THREAD

**Issue by bradarv90** [FredericHeem/redux-act-async#4]
I modified one of the tests in createActionAsyncTest.js to be 

```
it('should take 2 parameters', function () {
    let actionName = 'LOGIN_2';
    let user = {id: 8};
    let error = {name: 'myError'};

    function apiOk(userName, password){
      if (password === 'password' && userName === 'lolo')
        return Promise.resolve(user);
      return Promise.reject(error);
    }
    const login = createActionAsync(actionName, apiOk);
    const initialState = {
      authenticated: false,
    };

    let reducer = createReducer({
      [login.request]: (state, payload) => {
        console.log('login.request ', payload);
      },
      [login.ok]: (state, payload) => {
        console.log('login.ok ', payload);
      },
      [login.error]: (state, payload) => {
        console.log('login.error ', payload);
      }
    }, initialState);

    const store = createStore(reducer, applyMiddleware(thunk));
    let run = login('lolo', 'password');
    store.dispatch(run);
});
```

I would expect this promise to be resolved, but instead it's rejected since the second parameter comes in as undefined.

**Comment by FredericHeem** [comment]
For now, try to pass the username and password as a unique object until a proper fix is implemented. Thanks for the report

**Comment by FredericHeem** [comment]
solved in v0.4.2

**Comment by FredericHeem** [comment]
Actually, it wasn't solved in 0.4.2 but in 0.5.0
